src/extension/src/TelemetryWriter.py

```diff
@@ -27,11 +27,12 @@
 class TelemetryWriter(object):
     """Class for writing telemetry data to events"""
 
-    def __init__(self):
+    def __init__(self, logger):
+        self.logger = logger
         self.events_folder_path = None
-        self.operation_id = ""
+        self.__operation_id = ""
 
-    def __new_event_json(self, task_name, event_level, message):
+    def __new_event_json(self, event_level, message, task_name):
         return {
             "Version": Constants.EXT_VERSION,
             "Timestamp": str((datetime.datetime.utcnow()).strftime(Constants.UTC_DATETIME_FORMAT)),
```

Comment:
Don't restrict timestamp with a format, log time to the lowest unit available, will need it to sort telemetry in kusto